email.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Send email</title>
+    <script src="https://smtpjs.com/v3/smtp.js"></script>
+</head>
+<body>
+<script>
+Email.send({
+    Host : "smtp.elasticemail.com",
+    Username : "muskantewari1@hotmail.com",
+    Password : "3025c3e0-9741-47c0-a98d-ea6e38579a76",

```

Comment:
@muskantewari 

**Passwords are too sensitive data.** 
It's not supposed to be pushed on GitHub.

Can you please encrypt your SMTP credentials or remove the password?

```
Password : "XXXXXXXXXXXXX"
```

Refer [this](https://smtpjs.com/#useit).